teacher-technology-stack.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My teacher's technology</title>     
+    </head>
+    <body>
+        <h1>My teacher's technology stack</h1>
+        <ol>
+            <li>Ruby</li>
+            <li>Ruby on rails</li>
+            <li>Linux</li>
+            <li>JavaScript-Nodejs</li>
+            <li>Reactjs</li>
+        </ol>
+    </body>
+    </html>
```

Comment:
Now kuna kitu kinaitwa indentation. ili code zako ziwe katika good format **(CODE READABILITY)**

```
<!DOCTYPE html>
<html>
  <head>
    <title>My teacher's technology</title>     
  </head>
  <body>
    <h1>My teacher's technology stack</h1>
    <ol>
      <li>Ruby</li>
      <li>Ruby on rails</li>
      <li>Linux</li>
      <li>JavaScript-Nodejs</li>
      <li>Reactjs</li>
    </ol>
  </body>
</html> 
```

https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/spacing-indentation-html.html
https://stackoverflow.com/questions/34174207/how-to-change-indentation-in-visual-studio-code/45671704

When working with me @leonardk1 , I use two space indentation. learn what `two space` indentation is. it is good for you to make you code look good, and readable by others

Reply:
Check out these examples of how I use two space indentatation
```
def get_base_url(env, subdomain, api_path)
  url = case env
  when 'production'
    "https://#{subdomain}.my-portal.com/"
  when 'staging'
    "https://#{subdomain}.dev.home.com/"
  else
    "http://#{subdomain}.lvh.me:3000/"
  end

  url + api_path
end

def users(url)
  url = url + 'users'

  RestClient::Request.execute(method: :get,
                              url: url,
                              payload: {},
                              headers:{Authorization: "Bearer #{@token}"} )
end
```

Reply:
also you have not yet renamed the file, to reflect its purpose. the naming of our files also has to be meaningful

Reply:
true